### PR TITLE
feat: Add sidetree to vc-rest

### LINKED ISSUE #65: Add sidetree to vc-rest

### PR DESCRIPTION
**Pull request by fqutishat**
closes #65

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/internal/mock/kms/mock_kms.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package legacykms
```
Comment (sandrask) [inline pkg/internal/mock/kms/mock_kms.go]: We should expose certain mocks in aries so we can use them.

File: pkg/internal/mock/vdri/mock_registry.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+package vdri
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	vdriapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdri"
+)
+
+// MockVDRIRegistry mock implementation of vdri
+// to be used only for unit tests
+type MockVDRIRegistry struct {
```
Comment (sandrask) [inline pkg/internal/mock/vdri/mock_registry.go]: Same as above, we should expose mocks from aries.

File: test/bdd/fixtures/vc-rest/docker-compose.yml [hunk 3]
```diff
@@ -14,6 +14,7 @@ services:
     environment:
       - VC_REST_HOST_URL=${VC_HOST}:${VC_PORT}
       - EDV_REST_HOST_URL=${EDV_HOST}:${EDV_PORT}
+      - SIDETREE_HOST_URL=http://localhost:48326/document
```
Comment (rolsonquadras) [inline test/bdd/fixtures/vc-rest/docker-compose.yml]: may be we can move this config to .env file.

Comment (fqutishat) [reply test/bdd/fixtures/vc-rest/docker-compose.yml]: next PR

File: pkg/internal/mock/vdri/mock_registry.go (hunk 2 above)
Comment (fqutishat) [reply pkg/internal/mock/vdri/mock_registry.go]: done

File: pkg/internal/mock/kms/mock_kms.go (hunk 1 above)
Comment (fqutishat) [reply pkg/internal/mock/kms/mock_kms.go]: done

**Pull request merged**